ingests/shift.py

```diff
@@ -0,0 +1,357 @@
+"""Fetch data on fossil fuels energy production from The Shift Data Portal.
+
+"""
+
+import json
+import re
+import tempfile
+from pathlib import Path
+from time import sleep
+from typing import List
+
+import click
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from owid.datautils import dataframes
```

Comment:
`walden` does not have `datautils` installed, you'll have to add it with poetry.

Reply:
True, I just realised that. And datautils depends on owid-catalog... Is this going to cause issues?